src/main/webapp/js/const.es6

```diff
@@ -16,4 +42,98 @@ const Const = {
+                                                            'by the specified increment.',
+        DISPLAY_FEEDBACK_SESSION_VISIBLE_DATEINVALID: 'Feedback session visible date must not be empty',
+        DISPLAY_FEEDBACK_SESSION_PUBLISH_DATEINVALID: 'Feedback session publish date must not be empty',
+
+    },
+
+    // Used in instructorFeedbackEdit.js
+    InstructorFeedbackEdit: {
+
+        FEEDBACK_QUESTION_RECIPIENTTYPE: 'recipienttype',
+        FEEDBACK_QUESTION_NUMBEROFENTITIESTYPE: 'numofrecipientstype',
+        FEEDBACK_QUESTION_TYPE: 'questiontype',
+        FEEDBACK_QUESTION_MCQCHOICE: 'mcqOption',
+        FEEDBACK_QUESTION_MSQCHOICE: 'msqOption',
+        FEEDBACK_QUESTION_CONSTSUMOPTION: 'constSumOption',
+        FEEDBACK_QUESTION_CONSTSUMOPTIONTABLE: 'constSumOptionTable',
+        FEEDBACK_QUESTION_CONSTSUMTORECIPIENTS: 'constSumToRecipients',
+        FEEDBACK_QUESTION_CONSTSUMPOINTS: 'constSumPoints',
+        FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHOPTION: 'constSumPointsForEachOption',
+        FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHRECIPIENT: 'constSumPointsForEachRecipient',
+        FEEDBACK_QUESTION_NUMBEROFCHOICECREATED: 'noofchoicecreated',
+        FEEDBACK_QUESTION_NUMSCALE_MIN: 'numscalemin',
+        FEEDBACK_QUESTION_NUMSCALE_MAX: 'numscalemax',
+        FEEDBACK_QUESTION_NUMSCALE_STEP: 'numscalestep',
+        FEEDBACK_QUESTION_TEXT: 'questiontext',
+        FEEDBACK_QUESTION_DESCRIPTION: 'questiondescription',
+        FEEDBACK_QUESTION_EDITTEXT: 'questionedittext',
+        FEEDBACK_QUESTION_DISCARDCHANGES: 'questiondiscardchanges',
+        FEEDBACK_QUESTION_EDITTYPE: 'questionedittype',
+        FEEDBACK_QUESTION_SAVECHANGESTEXT: 'questionsavechangestext',
+        FEEDBACK_QUESTION_SHOWRESPONSESTO: 'showresponsesto',
+        FEEDBACK_QUESTION_SHOWGIVERTO: 'showgiverto',
+        FEEDBACK_QUESTION_SHOWRECIPIENTTO: 'showrecipientto',
+        FEEDBACK_QUESTION_TYPENAME_TEXT: 'Essay question',
+        FEEDBACK_QUESTION_TYPENAME_MCQ: 'Multiple-choice (single answer)',
+        FEEDBACK_QUESTION_TYPENAME_MSQ: 'Multiple-choice (multiple answers)',
+        FEEDBACK_QUESTION_TYPENAME_NUMSCALE: 'Numerical-scale question',
+        FEEDBACK_QUESTION_TYPENAME_CONSTSUM_OPTION: 'Distribute points (among options) question',
+        FEEDBACK_QUESTION_TYPENAME_CONSTSUM_RECIPIENT: 'Distribute points (among recipients) question',
+        FEEDBACK_QUESTION_TYPENAME_CONTRIB: 'Team contribution question',
+        FEEDBACK_QUESTION_TYPENAME_RUBRIC: 'Rubric question',
+        FEEDBACK_QUESTION_TYPENAME_RANK_OPTION: 'Rank options question',
+        FEEDBACK_QUESTION_TYPENAME_RANK_RECIPIENT: 'Rank recipients question',
+        FEEDBACK_QUESTION_RANKOPTION: 'rankOption',
+        FEEDBACK_QUESTION_RANKOPTIONTABLE: 'rankOptionTable',
+        FEEDBACK_QUESTION_RANKTORECIPIENTS: 'rankToRecipients',
+
+    },
+
+};
+
+// Needs instantiate Const first to process string interpolation
```

Comment:
Is there anything that stops the following constants to be put in the `Const` class directly?
```java
const Const = {
    ...
    InstructorCourse: {
        ...
    },
}
```
or, as per my previous comment:
```java
const Const = {
    ...
    ParamsNames: {
        ...
        COURSE_ID: 'courseid',
        ...
    },
}
```
Similarly for `Const.StatusType.isValidType`.